cargo-bolero/src/project.rs

```diff
@@ -70,12 +72,25 @@ impl Project {
                 cmd
             }
         };
+
         if self.rustc_bootstrap {
             cmd.env("RUSTC_BOOTSTRAP", "1");
+        } else if Self::toolchain_channel_is_stable() {
+            PRINT_BOOTSTRAP_WARNING.call_once(|| warning("detected a stable toolchain; the variable `RUSTC_BOOTSTRAP` will be set for all commands"));
+            cmd.env("RUSTC_BOOTSTRAP", "1");
         }
         cmd
     }
 
+    // Determines if the toolchain channel is stable
+    fn toolchain_channel_is_stable() -> bool {
+        let channel = std::env::var("RUSTUP_TOOLCHAIN");
```

Comment:
Is this the most reliable way to get the toolchain version? It also does mean that users not using rustup won't have this set.

Reply:
One option here is to use the [`rustc_version` crate](https://crates.io/crates/rustc_version). What do you think?

Reply:
Yeah that should work. That's what I was using before to auto-enable nightly. The crate was never removed even though the feature was: https://github.com/camshaft/bolero/blob/ec0ba25c6724f9b40c7130b4594a05eae5c09177/cargo-bolero/Cargo.toml#L27